contrib/i3status-rust/template.py

```diff
@@ -16,9 +21,16 @@
 maintainer = "ttyyls <contact@behri.org>"
 license = "GPL-3.0-only"
 url = "https://github.com/greshake/i3status-rust"
-source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
-sha256 = "fd722f630080ef0b25558bec9b342a9fe8842c3af049d55a1370c6760ad84c67"
+source = [
+    f"{url}/archive/refs/tags/v{pkgver}.tar.gz",
+    f"!https://dev.alpinelinux.org/archive/i3status-rs/i3status-rs-{pkgver}.1",
```

Comment:
this is not guaranteed to be stable or exist

Reply:
it gets updated when alpine updates their i3status package, assuming we do it in sync it should never cause any issues.

Reply:
other solutions could be to  

1. mirror it
2. build it ourselves using a similar setup
3. to simply drop it since a native build will be enabled anyway once a pandoc port is merged.

Reply:
i know, i'm the one that did that in the first place  

i would rather not rely on waiting for it tho. if upstreams don't want to ship a manpage without 'needing pandoc' then having no manpage at all is fine

Reply:
just dropped it.

Reply:
> to simply drop it since a native build will be enabled anyway once a pandoc port is merged.  

most likely it wouldn't be, because pandoc would be limited to where ghc is which is just a few architectures. actually having it everywhere is no chance